Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -274,6 +282,7 @@ AppDelegateViewModelOutputs {
     let deepLinkFromUrl = Signal
       .merge(
         openUrl.map { Navigation.match($0.url) },
+        self.foundRedirectUrlProperty.signal.skipNil().map(Navigation.match),
```

Comment:
once we find the redirect we can just feed it right into our deep linking infrastructure!